src/quests/keys.ts

```diff
@@ -0,0 +1,60 @@
+import { availableAmount, cliExecute, restoreMp, useSkill } from "kolmafia";
+import {
+  $item,
+  $items,
+  $location,
+  $skill,
+  adventureMacro,
+  get,
+  have,
+  maximizeCached,
+  sum,
+} from "libram";
+
+import { Macro } from "../combat";
+import { propertyManager } from "../global";
+import { acquire } from "../lib";
+import { moodBaseline } from "../mood";
+
+function keyCount() {
+  return sum($items`fat loot token, Boris's key, Jarlsberg's key, Sneaky Pete's key`, (item) =>
+    availableAmount(item)
+  );
+}
+
+export function heroKeys(): void {
+  if (have($item`Deck of Every Card`) && !get("_deckCardsSeen").includes("Tower")) {
+    cliExecute("play Tower");
+  }
+
+  if (have($skill`Lock Picking`) && !get("lockPicked")) {
+    propertyManager.setChoices({
```

Comment:
I often create a setChoice function for individual choices. Maybe I should just make a method.

Reply:
we should just add that to libram - I like having it on the manager explicitly